docs: change lite-server baseDir path to output folder

ngdevelop-tech opened the pull request:
lite-server baseDir path should be the output folder path, So need to change `baseDir="dist"` to `baseDir="dist/project-name"

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Current shell command to deploy application on lite-server is `lite-server ---baseDir="dist"`, this command is not able to find the project output folder files. 
Issue Number: N/A


## What is the new behavior?
project work properly in lite-server

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

ngdevelop-tech committed:
docs: change lite-server baseDir path to output folder

In lite-server baseDir, path should be the output folder path, So need to change `baseDir="dist"` to `baseDir="dist/project-name"`

ngdevelop-tech commented:
@googlebot I signed it!

kapunahelewong commented:
Hi @ngdevelop-tech, thank you for this fix. Could you update your commit message? That's the reason that one test is failing. Try something like:

```
docs: change lite-server baseDir path to output folder
```

That should help. Feel free to tag me when you're ready.

ngdevelop-tech commented:
Hi @kapunahelewong, Thank you for the review, I have changed the commit message. 
Please let me know if there are any changes required.

kapunahelewong commented:
Thanks, @ngdevelop-tech! It looks like those failures are flakes but rebasing might fix them. Once you've rebased, please let me know. 🌸

ngdevelop-tech commented:
Hi @kapunahelewong, I have rebased the branch, but circleci checks doesn't get triggered and it is still showing errors and failing. Please let me know what changes I need to do further to fix this.
Please help.

ngdevelop-tech commented:
Hi @kapunahelewong, I have rebased the branch, I don't know whether is a proper way because everyone's commit is now visible here, I am not that strong in git, can you please help me 😔 ?,

I have executed the following commands : 
First I have changed the commit message using `git commit --amend`
Then I have rebased using `git rebase upstream/master`
because it was showing one pull so I have executed: `git pull -r`.
and then I pushed using `git push`

I don't know it is proper or not, please guide me further what steps I have to take. 
Sorry for the inconvenience 😔

kapunahelewong reviewed (approved):
LGTM 😊

Thank you for this, @ngdevelop-tech 🌟

ngdevelop-tech commented:
Hi @kapunahelewong, I have followed the workflow you mentioned, now all checked are passed. 
Thank you for your guidance. It helped me a lot to learn new things,

ngdevelop-tech commented:
Thank you @kapunahelewong , 
It was a great learning experience 🙂.
